### ISSUE TITLE
Allow deleting custom task type

### ISSUE THREAD

**Issue by siew24** [hcengineering/platform#7194]
### Description of the issue
I could not delete a custom task type even though no issue was referencing it.

### Your environment
* Version of Huly: 0.6.343 (0.6.343)
* Browser (and version): Huly Desktop
* Your operating system (and version): MacOS Sequoia 15.1

### Steps to reproduce
1) Create a custom task type in Classic Project space settings
2) Try and delete it.

### Expected behaviour
The custom task type should be able to be deleted when no issues are referencing it.

### Actual behaviour
The custom task type is not able to be deleted. Even if so, there's no deletion UI for it.
<img width="1032" alt="Screenshot 2024-11-19 at 1 47 00 PM" src="https://github.com/user-attachments/assets/478978f8-c830-42a3-bdb9-bbe197ccef23">

**Comment by ntudor11** [comment]
Agreed - this is highly useful especially when setting up a project and tweaking different settings to test what best suits a team. Please advise whether this is going to be included in the roadmap.
